### ISSUE TITLE
Add support for the updated version of Crunchyroll Downloader Toolkit DX

### ISSUE THREAD

**Issue by Darken-kun** [GameMasterJack/Crunchyroll-Anime-Downloader-UI#3]
Hello, I would like to ask if there's a way to make this add-on you created compatible with the latest version of the Crunchyroll Downloader Toolkit DX. Here's the link of it: https://goo.gl/CDFVQF

 Meanwhile the version of the program you created is based on a 2013 build, this one is still nowadays receiving updates, besides being the only one which actually works in my PC...

 Of course, I already tried to just run the add-on in the latest build but it gives errors messages because lots of files were modified since then. By the way, I don't understand anything of programing so I don't have a glimpse if whether this is possible or not, and that's why I'm asking this.

**Comment by GameMasterJack** [comment]
Hey Bandido, I havn't really worked on it in quite a while. However if you
want I could take a look and rewrite it, I don't know how long it would
take though. Maybe a couple weeks.

**Comment by Darken-kun** [comment]
Yes, I would really apreciate it. Since there are times when I want to download the full season of an anime instead one episode at a time, this tool would actually be a great help. If you think it's possible, just take your time, I don't mind waiting at all.

> Hey Bandido, I havn't really worked on it in quite a while. However if you
> want I could take a look and rewrite it, I don't know how long it would
> take though. Maybe a couple weeks.

**Comment by GameMasterJack** [comment]
Alright, I'll take a look at it tomorrow.

**Comment by Darken-kun** [comment]
Hey, have you made any progress since then?

**Comment by GameMasterJack** [comment]
Yeah I have, I have everything ready but it appears there is something
wrong with the base program itself. As it's throwing parsing errors. I took
a break on it because it was annoying the shit out of me. I'll take another
look however.

**Comment by Darken-kun** [comment]
Well, it seems they did some major updates in the Toolkit in the last couple weeks. Now they added a new option called "Autocatch links", where you can paste the Crunchyroll link of the anime you want and it'll download all the episodes avaliable. 

 It's still not as pratical as the add-on you created since you can't choose exactly which episodes to download, but that's the thing I was needing all along. But anyway, I thank you for your will to help and effort of changing the codes of your program.

**Comment by vampjoseph** [comment]
I don't know what the OP is talking about and would still like an updated version of the UI if you haven't worked on it or released it already.

**Comment by Darken-kun** [comment]
> I don't know what the OP is talking about and would still like an updated version of the UI if you haven't worked on it or released it already.

@vampjoseph What I was saying is that now there is a much better version of the Crunchyroll Downloader avaliable on GitHub and this add-on here is no longer necessary. In the latest version, you can already download an entire anime season all by once, besides having much less bugs than the older version. If you're interested, you can download it in this link: https://github.com/jaw20/Crunchyroll-XML-Decoder

By the way, sorry for taking so long to reply. I only saw your comment today.
